clients/lsp-rust.el

```diff
@@ -585,6 +585,13 @@ The command should include `--message=format=json` or similar option."
 (lsp-defun lsp-rust--analyzer-run-single ((&Command :arguments?))
   (lsp-rust-analyzer-run (lsp-seq-first arguments?)))
 
+;; `lsp-interface' is unsuitable for ShowReference destructuring, as that code
+;; action yields an array of three elements, not a map.
```

Comment:
lsp-interface is for defining destructuring for objects, not for arrays. dash has destructuring for arrays/lists thus there is no need for lsp-interface to handle that. Thus, this comment can be deleted.

Reply:
Will do that.

Reply:
Fixed.